app/views/settings/index.slim

```diff
@@ -121,7 +121,7 @@
                                   span#copied style='display: none;' = icon("fas", "check")
                           td
                             - if (Time.now - 24.hours) > application.created_at
-                              i Hidden forever
+                              button.btn.btn-secondary.btn-block disabled=true Hidden forever
```

Comment:
I lied! 2 more things!  Their should probably not be any space between the disabled block and the X button.

Reply:
The X button deletes the application so it belongs to the whole row, not to that column, so I think it's correct that they're separate.

<img width="387" alt="screenshot 2018-06-11 10 06 51" src="https://user-images.githubusercontent.com/438911/41246225-42d17de4-6d5f-11e8-8907-e4a02bdbe054.png">

Though I admit that's not obvious as a user. I will try to visually differentiate it a bit more.

Reply:
Ah, that makes much more sense then.   I thought it would just reset the client secret so that you could get a new one if you llwt the old one.  Is there not a way to do that?

Reply:
No, you have to make a new client. I'll make a ticket to allow resetting since most providers let you do that.